File: java_backup/my java/souvik/fibonacci.java
class fibonacci
{
public static void main(int n)
{
int i,a,b,c;
a=0;
b=1;
System.out.println(a+",");
System.out.println(b+",");
for(i=1;i<=n-2;i++)
{
c=a+b;
System.out.println(c+",");
a=b;
b=c;
}
}
}

File: python/python_new/Python 3/fibonacci.py
nterms=10
#A = int(input("How many terms? "))
n1=0
n2=1
count=0
if nterms <= 0:
 print("Enter a positive integer")
elif nterms == 1:
 print("Fibonacci sequence upto",nterms,":")
 print(n1)
else:
 print("Fibonacci sequence upto",nterms,":")
while count < nterms:
 print (n1,end=' , ')
 n3=n1+n2
 #update values
 n1=n2
 n2=n3
 count+=1
